src/components/home/Team/index.js

```diff
@@ -0,0 +1,75 @@
+import _ from "lodash"
+import Img from "gatsby-image"
+import React from "react"
+
+import { StaticQuery, graphql } from "gatsby"
+
+import "./index.module.css"
+
+const socialLink = (platform, username) => {
+  switch (platform) {
+    case "twitter":
+      return `https://twitter.com/${username}`
+    default:
+      return "https://subvisual.co"
```

Comment:
Why this default? Doesn't it make more sense to simply return `#`?

Reply:
Why? For accessibility it's better to have a valid link.